app/utils/expandable-record-array.js

```diff
@@ -53,7 +53,7 @@ export default ArrayProxy.extend({
     return results;
   },
 
-  observedArraydidChange(array, index, removedCount, addedCount) {
+  observedArrayDidChange(array, index, removedCount, addedCount) {
```

Comment:
Depending on how you do things, this could be an API change. I'm assuming it isn't.